test/performance/specs/post-editor.spec.js

```diff
@@ -657,7 +657,7 @@ test.describe( 'Post Editor Performance', () => {
 
 				const startTime = performance.now();
 
-				await page.getByRole( 'button', { name: 'Test' } ).click();
+				await page.getByText( 'Test' ).click();
```

Comment:
This locator was causing the failure. The element is button on `wp/6.5` branch, but after recent refactoring it's `tab` on trunk. See #60257.